frontend/assets/stylesheets/components/_payment-b.scss

```diff
@@ -0,0 +1,261 @@
+.skin-checkout-b {
+    padding: 0;
+    width: 100%;
+    z-index: 500;
+
+    background: #ffffff;
+
+    .form-header {
+    	margin-top: 6px;
+    }
+
+    .form-header,
+    .form-section__content {
+        float: none;
+        margin-right: 10px;
+        margin-left: 10px;
+        width: auto;
+        padding: 0;
+
+        @include mq($from: mobileLandscape) {
+            max-width: 440px;
+            margin-left: auto;
+            margin-right: auto;
+        }
+
+        @include mq($from: tablet) {
+            max-width: 700px;
+            margin-left: auto;
```

Comment:
`margin-left` and `margin-right` are unnecessary here as they are specified in `$from: mobileLandscape` breakpoint